glustercli/cmd/volume.go

```diff
@@ -218,6 +219,9 @@ var volumeDeleteCmd = &cobra.Command{
 	Args:  cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
 		volname := cmd.Flags().Args()[0]
+		if ok := prompt.Confirm("Are you sure you want to delete volume %s [yes/no]", volname); !ok {
```

Comment:
question: why can't we implement this functionality in glusterd2 instead of importing external package for this?

Reply:
this doesn't require much functionality, we can easily implement this.